SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp

```diff
@@ -151,6 +154,7 @@ void ASpatialDebugger::BeginPlay()
 		{
 			SpatialToggleDebugger();
 		}
+		WireFrameMaterial = LoadObject<UMaterial>(nullptr, *DEFAULT_WIREFRAME_MATERIAL);
```

Comment:
If this is null, will it crash when setting it later as a material? Either way, might be good to error/warn if it's null.

Reply:
Added a null check and log warning.